http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppServiceDbQueriesSpec.scala

```diff
@@ -1,442 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo
```

Comment:
I moved all tests in this file to `KubernetesServiceDbQueriesSpec`, since that is the class being tested (there is no class named `AppServiceDbQueries`)